ui/perfherder/App.jsx

```diff
@@ -87,6 +91,7 @@ class App extends React.Component {
                     user={user}
                     projects={projects}
                     frameworks={frameworks}
+                    performanceTags={performanceTags}
```

Comment:
I think we've forgotten to guard these routes by uninitialized `performanceTags`.
Currently, they're guarded by this condition:
```javascript
{projects.length > 0 && frameworks.length > 0 && (
  <route configs>
}
```